README.md

```diff
@@ -4,25 +4,30 @@ Twig is a browser-based single-page application using [D3](https://d3js.org/) to
 
 ## Table of Contents
 
-* [Quick Start for Twig](#quick-start)
-* [Quick Start for Twig API](#quick-start-for-twig-api)
-* [Why Twig?](#why-twig)
-* [What Can Twig Do?](#what-can-twig-do)
-* [Demo](#demo)
-* [Where is Twig Deployed?](#where-is-it-deployed)
-* [Getting Started](#getting-started)
-  * [Development](#development)
-  * [Build](#build)
-  * [Unit Tests](#unit-tests)
-  * [End-to-End Tests](#end-to-end-tests)
-  * [API Documentation](#twig-api-documentation)
-* [User Notes](#user-notes)
-  * [How To Contribute](#how-to-contribute)
-  * [Tools](#tools)
-  * [Team](#team)
-  * [License](#license)
-  * [Further Help](#further-help)
-* [Living Style Guide](https://buildit.github.io/twig/)
```

Comment:
Was there a reason this link to the Living Style Guide was removed? The link still looks good